src/internal/kindenv/kindenv.go

```diff
@@ -227,6 +230,9 @@ func (c *Cluster) GetConfig(ctx context.Context, namespace string) (*ClusterConf
 	} else {
 		errors.JoinInto(&errs, errors.New("unable to determine whether or not to use TLS"))
 	}
+	if console, ok := ns.Annotations[consoleOverrideKey]; ok {
```

Comment:
`ok && console != ""`?

Reply:
result.ConsoleTag is already "" (the zero value for a string), so we wouldn't change the outcome of the function if we added that check 😂